spec/features/admin/budgets_spec.rb

```diff
@@ -126,6 +126,33 @@
 
       expect(page).to have_content "New participatory budget created successfully!"
       expect(page).to have_content "M30 - Summer campaign"
+
+      budget = Budget.order(:id).last
+      expect(budget.knapsack_voting?).to be true
+    end
+
+    scenario "Create budget - voting style approval", :js do
```

Comment:
@voodoorai2000 Is this ok, or will it be good to split it in to some contexts.

```
context "New" do
  context "Approval" do
    scenario "Create budget"
  end

  context "Knapsack" do
    scenario "Create budget"
  end

  // ...
end
```
I have flattened it out because the other budget actions are not effected by if it is approval or knapsack.

```
context "New" do
  scenario "Create budget - Knapsack"
  scenario "Create budget - Approval"
  // ...
end
```

Reply:
Sure that's fine 👍